examples/external_aerodynamics/domino/data_sources.py

```diff
@@ -152,135 +111,28 @@ def read_file(self, dirname: str) -> DoMINOExtractedDataInMemory:
             reader = vtk.vtkXMLUnstructuredGridReader()
```

Comment:
Consider using `pyvista` for reading vtu files instead of VTK, same as vtp. I think pv should be able to handle it.
Does not have to be in this PR.